code/game/objects/items/toys.dm

```diff
@@ -1032,45 +1032,81 @@
 	var/cooldown = 0
 
 /obj/item/toy/nuke/attack_self(mob/user)
-	if (cooldown < world.time)
-		cooldown = world.time + 1800 //3 minutes
+	var/timeleft = (cooldown - world.time)
```

Comment:
timeleft is only used at the else so move it there

Reply:
Fixed.